docs/contribute/contribution_gallery.rst

```diff
@@ -0,0 +1,152 @@
+===========================
+Contributing to the Gallery
+===========================
+
+Get Started
+-----------
+
+Ready to contribute? Here's how to set up `pyfar_gallery` for local development using the command-line interface. Note that several alternative user interfaces exist, e.g., the Git GUI, `GitHub Desktop <https://desktop.github.com/>`_, extensions in `Visual Studio Code <https://code.visualstudio.com/>`_ ...
+
+1. `Fork <https://docs.github.com/en/get-started/quickstart/fork-a-repo/>`_ the `gallery` repo on GitHub.
+2. Clone your fork locally and cd into the gallery directory::
+
+.. code-block:: shell
+
+    git clone https://github.com/YOUR_USERNAME/gallery.git
+    cd gallery
+
+3. Install your local copy into a virtualenv. Assuming you have Anaconda or Miniconda installed, this is how you set up your fork for local development::
+
+.. code-block:: shell
+
+    conda create --name gallery python
+    conda activate gallery
+    pip install -r requirements.txt
+
+4. Set up pre-commit hooks. This will cause commits to fail and clean all notebooks if the notebooks in `docs/gallery/interactive` aren't cleaned from outputs. After that, the automatic changes can be added and committed::
```

Comment:
```suggestion
4. Set up pre-commit hooks. This will cause commits to fail and clean all notebooks if the notebooks in `docs/gallery/interactive` aren't cleaned from outputs. After that, the automatic changes can be added and committed:
```